README.md

```diff
@@ -0,0 +1,30 @@
+# rename-mongodb-database
+<a href="https://apostrophecms.com/"><img src="https://raw.githubusercontent.com/apostrophecms/rename-mongodb-database/main/logos/logo-box-madefor.png" align="right" /></a>
```

Comment:
Remove this badge, this is not our brand.

I know you want to formalize this but for now just use a text link to apostrophecms.com